scripts/zones/Mhaura/npcs/Nereus.lua

```diff
@@ -3,61 +3,60 @@
 --  NPC: Nereus
 -- Starts and ends repeteable quest A_POTTER_S_PREFERENCE
 -----------------------------------
-
---     player:startEvent(110); standar dialog
+package.loaded["scripts/zones/Mhaura/TextIDs"] = nil
+-----------------------------------
+require("scripts/zones/Mhaura/TextIDs")
+require("scripts/globals/settings")
+-----------------------------------
+--    player:startEvent(110); -- standar dialog
 --    player:startEvent(115); -- i have enough for now, come later
---    player:startEvent(114); --  get me x as soon as you can
+--    player:startEvent(114); -- get me x as soon as you can
 --    player:startEvent(111); -- start quest A Potter's Preference
 --    player:startEvent(113); -- quest done!
 --    player:startEvent(112); -- repeat quest
 
 function onTrade(player,npc,trade)
-if ((player:getQuestStatus(OTHER_AREAS_LOG,A_POTTER_S_PREFERENCE) == QUEST_ACCEPTED) or (player:getVar("QuestAPotterPrefeRepeat_var")==1)) then
-    local gusgenclay  = trade:hasItemQty(569,1);
-    if (gusgenclay  == true) then
-        player:startEvent(113); -- quest done!
-    end;
-end;
-end;
+    if player:getQuestStatus(OTHER_AREAS_LOG,A_POTTER_S_PREFERENCE) == QUEST_ACCEPTED or player:getVar("QuestAPotterPrefeRepeat_var") == 1 then
```

Comment:
script looks like it should have quest require

Reply:
k